dotCMS/src/main/java/com/dotmarketing/business/UserAPI.java

```diff
@@ -91,6 +91,8 @@
 	 */
 	public List<User> getUsersByName(String filter, int start,int limit, User user, boolean respectFrontEndRoles) throws  DotDataException;
 
+	public long getCountUsersByName(String filter) throws DotDataException;

```

Comment:
The name All the others "count methods" in this class are:

- public long getCountUsersByNameOrEmailOrUserID(String filter) throws DotDataException;
- public long getCountUsersByNameOrEmailOrUserID(String filter, boolean includeAnonymous) throws DotDataException;